packages/core/package.json

```diff
@@ -3,22 +3,20 @@
   "version": "4.2.0",
   "description": "A simple React component capable of building HTML forms out of a JSON schema.",
   "scripts": {
-    "build-umd": "cross-env NODE_ENV=production BABEL_ENV=umd webpack --config webpack.config.dist.js",
-    "build": "rimraf dist && tsdx build --format cjs,es && npm run build-umd",
-    "cs-check": "prettier -l \"{src,test}/**/*.js\"",
-    "cs-format": "prettier \"{src,test}/**/*.js\" --write",
+    "build": "rimraf dist && tsdx build --format cjs,es,umd",
+    "cs-check": "prettier -l \"{src,test}/**/*.[jt]s?(x)\"",
+    "cs-format": "prettier \"{src,test}/**/*.[jt]s?(x)\" --write",
     "lint": "eslint src test",
     "precommit": "lint-staged",
     "publish-to-npm": "npm run build && npm publish",
     "start": "tsdx watch",
     "test": "tsdx test",
     "test:watch": "tsdx test --watch",
     "test:update": "tsdx test --u",
-    "test-old": "cross-env BABEL_ENV=test NODE_ENV=test mocha --require @babel/register --require ./test/setup-jsdom.js test/**/*_test.js",
     "test-coverage": "tsdx test --coverage"
   },
   "lint-staged": {
```

Comment:
Does the lint-staged rule have to be here? Can we move it to the base package.json and remove it from all sub-packages for consistency/single point of control?

Reply:
I'm not sure... I do know that different packages have different globs for the files to check so, for the moment I think we need to keep them per package